packages/core/src/button/useButton.ts

```diff
@@ -82,7 +82,7 @@ export const useButton = <T extends Element>({
   };
 
   const buttonProps = {
-    "aria-disabled": disabled,
+    "aria-disabled": focusableWhenDisabled,
```

Comment:
Not sure this is right, I would expect aria-disabled to be on when disabled and focusWhenDisabled is true